pkg/cloud/gcp/actuators/machine/actuator_test.go

```diff
@@ -215,8 +215,6 @@ func TestActuatorEvents(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
-			gs := NewWithT(t)
```

Comment:
We use a separate `gomega` so that when the test fails, it's only the subtest, not the whole test. Please revert removing this and the renaming of the previous `g` declaration as well